etcdctl/ctlv3/command/elect_command.go

```diff
@@ -73,7 +74,7 @@ func observe(c *clientv3.Client, election string) error {
 
 	donec := make(chan struct{})
 	sigc := make(chan os.Signal, 1)
-	signal.Notify(sigc, os.Interrupt, os.Kill)
+	signal.Notify(sigc, os.Interrupt, syscall.SIGTERM)
```

Comment:
should we be consistent on what which pkg to use for signal?  use `syscall` for all signal?